### PR TITLE
VMware: add new module : vcenter_folder

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/vmware/vcenter_folder.py
test/integration/targets/vcenter_folder/aliases
test/integration/targets/vcenter_folder/tasks/main.yml

##### ANSIBLE VERSION

```
2.5devel
```

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: lib/ansible/modules/cloud/vmware/vcenter_folder.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+EXAMPLES = r'''
+- name: Create a VM folder on given datacenter
+  vcenter_folder:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    datacenter: datacenter_name
+    folder_name: sample_vm_folder
+    folder_type: vm
+    operation: present
```
Comment (tchernomax) [inline lib/ansible/modules/cloud/vmware/vcenter_folder.py]: operation → state

Comment (tchernomax) [inline lib/ansible/modules/cloud/vmware/vcenter_folder.py]: operation → state

**Review by tchernomax — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/vmware/vcenter_folder.py (hunk 1 above)
Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vcenter_folder.py]: my bad.

Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vcenter_folder.py]: Done.

**Review by tchernomax — APPROVED** [review]
LGTM

**Comment by Akasurde** [comment]
@cvanniekerk @tchernomax Thanks for your valuable feedback and reviews.

**Pull request merged**